### ISSUE TITLE
It's not obvious to users that they can search within a product

### ISSUE THREAD

**Issue by jericson-edb** [EnterpriseDB/docs#1874]
I've heard from others and noticed myself that it isn't always obvious that search can be limited to a specific product. For instance, if I just type `install`into the search box, I see:

![image](https://user-images.githubusercontent.com/83660216/134540068-5d4e30e9-1397-49b5-8756-6433bb1b78b8.png)

The list of results don't show the product I want to install unless I happen to be looking for the open source PostgreSQL. There is an "Advanced Search" link which does exactly what I need, but I often don't see the link. Once people (including me) get to the advanced search, we can see exactly what we need. But the path to get there isn't clear.

Possible solutions:

1. Make the "Advanced Search" link more obvious. (This assumes the problem is people don't see it.)
2. Change the copy from "Advanced Search" to "Search by product". (This assumes the problem is people don't grok that "Advanced Search" is the feature they want. It also depends on no other advanced search features added later.)
3. Search the current product by default. (But then there will be a question about how people will get to the cross-product search.)
4. When people press "enter" in the search box, send them to the advanced search page with their search terms pre-populated. (Maybe just me, but I find myself pressing "enter" in hopes of getting a separate page with all the results rather than trying to scroll through the dropdown. Also, clicking "Advanced Search" doesn't populate the search terms and I think it should.)
5. Shorten the size of the dropdown so that people see the "Not finding what you need? Try Advanced Search" message sooner. (I lose patience with the dropdown after about 5 unhelpful results.)

I think any one of these would help and some combination would solve the problem.

**Comment by drothery-edb** [comment]
When researching solution 2 I noticed the Try Advanced Search link that appears when a site-wide search fails. The link makes no sense to me. If the global search fails, how can the user get better results if they search by product? Instead of having this a link to the product search page, I think we should say something like "Try another search term" and remove the link.

**Comment by jericson-edb** [comment]
> If the global search fails, how can the user get better results if they search by product? Instead of having this a link to the product search page, I think we should say something like "Try another search term" and remove the link.

I agree. Currently there's nothing in the advanced search that will help. Perhaps better would be to document our search and point people to that. A place to start is the [developer-facing README for search](https://github.com/EnterpriseDB/docs/tree/develop/docs/overviews/search). So it might be helpful to point out that including quotes [disables typo tolerance withing the quotes](https://www.algolia.com/doc/api-reference/api-parameters/advancedSyntax/). That sort of information could help people build a better search query.
